README.md

```diff
@@ -11,25 +12,155 @@ This operator is in charge to configure the idp client service.
 
 `make functional-test-full`
 
-# Deploy the operator
+# Installing
 
-1. Login on your hub
-2. From the idp-mgmt-operator: `make deploy`
+## Prereqs
+You must meet the following requirements:
+- OpenShift Container Platform (OCP) 4.6+
+- `oc` (ver. 4.6+)
 
-# Example
+## Ensure you are logged in to the correct OCP hub
+```bash
+oc cluster-info
+```
+
+## Setup Hub to use a signed certificate
+This is required for the Dex server we will be using under the covers to authenticate
+the managed clusters OpenID OAuth requests.  The quickest way to do this is using a tool from
+https://github.com/open-cluster-management/sre-tools/wiki/ACM---Day-1#add-an-acme-certificate.  Here is a summary of the commands you need to run:
+
+```bash
+export AWS_ACCESS_KEY={your AWS Key}
+export AWS_SECRET_ACCESS_KEY={your AWS Secret Key}
+export MY_EMAIL_ADDR={your email address}
+
+cd /tmp
+git clone https://github.com/acmesh-official/acme.sh.git
+cd acme.sh
+
+export API=$(oc whoami --show-server | cut -f 2 -d ':' | cut -f 3 -d '/' | sed 's/-api././')
+export WILDCARD=$(oc get ingresscontroller default -n openshift-ingress-operator -o jsonpath='{.status.domain}')
+
+
+./acme.sh --register-account -m ${MY_EMAIL_ADDR}
```

Comment:
This should be done only ones for you email address

Reply:
Yes, I was assuming a fresh clone.  But I will reformat to show this.